.github/workflows/ci.yaml

```
@@ -384,6 +384,20 @@ jobs:
             - name: Check
               run: cargo check --target=${{matrix.target}} -p printerdemo_mcu --no-default-features --features=mcu-board-support/${{matrix.feature}} --release
 
+    # test to compile the mcu backend for the arm target (no_std) using embassy
+    mcu-embassy:
+        env:
+            SLINT_FONT_SIZES: 8,11,10,12,13,14,15,16,18,20,22,24,32
+            RUSTFLAGS: --cfg slint_int_coord -D warnings
+            CARGO_PROFILE_DEV_DEBUG: 0
+            CARGO_PROFILE_RELEASE_OPT_LEVEL: s
+        runs-on: ubuntu-22.04
+        steps:
+            - uses: actions/checkout@v4
+            - uses: ./.github/actions/setup-rust
+            - name: Check
+              run: cargo check -p mcu_embassy --bin ui_mcu --target=thumbv8m.main-none-eabihf --no-default-features --features="mcu-embassy/mcu" --release
```

For reasons I don't quite understand, the CI didn't run on the latest. But I ran it manually and noticed this error:

```
error: cannot specify features for packages outside of workspace
```

I tried locally, could reproduce it, and I think this is the easy fix:

```suggestion
            - name: Check
              working-directory: examples/mcu-embassy
              run: cargo check --bin ui_mcu --target=thumbv8m.main-none-eabihf --no-default-features --features="mcu-embassy/mcu" --release
```